docs/guide/siteaccess.md

```diff
@@ -96,6 +96,28 @@ ezpublish:
 
     A new SiteAccess is recognized by the system, but an Anonymous User will not have read access to it until it is [explicitly given via the Admin > Roles panel](permissions.md#use-cases). Without read access the Anonymous User will simply be directed to the default login page.
 
+### Defining SiteAccess name
+
+In order to simplify the interface and create a better editorial experience, you can "hide"
+ the SiteAccess code and substitute it with a human readable name of the website e.g. `Tastefull Planet`. 
+ 
+List of interfaces where you can apply SiteAccess names: 
+
+- Page Builder (SiteAccess switcher in the top navigation) 
+
+- [Content Preview](https://doc.ezplatform.com/projects/userguide/en/latest/creating_content_basic/#previewing-content) (SiteAccess switcher in the dropdown menu) 
+
+- Page creation modal window (when coming from Content Structure) 
+
+You can also translate SiteAccess names. Displayed names depend on the selected language of the administration interface.
+ 
+You can define name and translation of the SiteAccess (both used as the keys) in `app/Resources/translations/ezplatform_siteaccess.en.yml`:
```

Comment:
translations must be put in yaml file with correct language code, don't know if this is clear enough. Maybe we should use different siteaccess codes (not language related) to keep this example more clean?